qa/rpc-tests/maxuploadtarget.py

```diff
@@ -192,7 +192,8 @@ def run_test(self):
         getdata_request.inv.append(CInv(2, big_old_block))
 
         max_bytes_per_day = 200*1024*1024
-        max_bytes_available = max_bytes_per_day - 144*1000000
+        daily_buffer = 144 * 1000000 # 144 * MAX_BLOCK_SIZE
```

Comment:
Is this exposed via rpc? I always like to do `grep -r 'CONSTANT' bitcoin/` when touching a constant to see it's usage.